services/minespace-web/src/components/layout/HeaderDropdown.js

```diff
@@ -74,10 +80,10 @@ export class HeaderDropdown extends Component {
     return (
       <span>
         <MediaQuery minWidth={smallestDesktopWidth}>
-          <Link to={route.MINES.route} className="header-link">
+          <Link to={route.MINES.route} className={this.setActiveLink("/mines")}>
             My Mines
           </Link>
-          <Link to={route.USERS.route} className="header-link">
+          <Link to={route.USERS.route} className={this.setActiveLink("/users")}>
```

Comment:
Should import routes and use MINES.route and USERS.route respectively

Reply:
yes good catch